docassemble/ALToolbox/data/questions/al_income.yml

```diff
@@ -725,6 +725,31 @@ subquestion: |
 fields:
   - Job name: x[i].source
 ---
+generic object: ALJobList
+continue button field: x.revisit
+question: |
+  Edit jobs
+subquestion: |
+  ${ x.table }
+
+  ${ x.add_action() }
+---
+generic object: ALJobList
+table: x.table
+rows: x
+columns:
+  - Title: |
+      row_item.source
+  - Employer: |
+      row_item.employer.name if hasattr(row_item.employer.name, "first") else ""
+  - Annual Income: |
+      currency(row_item.net_total())
+edit:
+  - source
+  - employer.name.first
+  - net
```

Comment:
Looking back, I chose `net` because I was using this class in the ALAffidavitOfIndigency, for jobs of other members in your household, and the form specifically says to "list occupation and name and address of his/her employer and monthly income after taxes".  `net` matched that better, so I went with it, but I agree with our conversation on Monday that `net` should be calculated, and there should instead be a `value` (gross) and `deductions` attribute. That's a bit bigger of a change (and does break the AffidavitOfIndigency), so I'll make it a different PR and just change this to `value` for now.

Reply:
These changes are in #144 now, should have addressed the comment